proto/tests/test_proto_fe.cpp

```diff
@@ -1530,7 +1505,19 @@ TEST_F(DirectCounterTest, InvalidRequestReadAllFromDefault) {
 
 // TODO(antonin)
 TEST_F(DirectCounterTest, ReadAllFromTable) {
-  auto counter_entry = make_counter_entry(nullptr);  // default TableEntry
+  std::string mf("\xaa\xbb\xcc\xdd", 4);
```

Comment:
why do we need to add an actual table entry to test this?

Reply:
We don't. I thought I will keep it like this so it will easier to test when we actually implement it.